src/App.tsx

```diff
@@ -1,61 +1,116 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
The `usersFromServer` and `todosFromServer` data should be prepared in the App component and passed down as props to child components. This ensures data consistency and maintainability. Consider refactoring to pass the necessary data as props.